### PR TITLE
Add --private wiki flag

### PR DESCRIPTION
**Pull request by featherware**

### CODE REVIEW

File: pkc [hunk 1]
```diff
@@ -268,7 +272,8 @@ run_setup() {
                         --wallet-address "$WALLET_ADDRESS" \
                         --pkc-server "$PKC_SERVER" \
                         --eauth-server "$EAUTH_SERVER" \
-                        ${empty_wiki:+--empty-wiki} || true
```
Comment (rht) [inline pkc:271]: The reason why the `|| true` is needed is for the case when `LocalSettings.php` already exists, and so the install.php exits with status 1. Adding `|| true` simply continues with the rest of the setup, i.e. seeding Eauth. Sorry that I haven't documented this. You can add a comment about this `|| true` in this PR.

Comment (Luan-vP) [reply pkc:271]: No problem, tbh thought it was do do with the variable substitution

File: aqua/extraAquaSettings.php [hunk 2]
```diff
@@ -63,4 +62,4 @@ function ModifyDefaultLogin(&$personal_urls, &$wgTitle) {
 
 $wgOAuth2Client['configuration']['scopes'] = 'openid email profile'; //Permissions
 $wgOAuth2Client['configuration']['service_login_link_text'] = 'Login with Ethereum wallet'; // the text of the login link
-$wgWhitelistRead = ['Special:OAuth2Client', 'Special:OAuth2Client/redirect'];
+$wgWhitelistRead = [ 'Main Page', 'Special:OAuth2Client', 'Special:Oauth2Client/redirect' ];
```
[the remark this answers is not in the document]
Comment (Luan-vP) [reply aqua/extraAquaSettings.php:65]: ^ just confirmed yes on a call 👍

[the remark this answers is not in the document]
Comment (rht) [reply aqua/extraAquaSettings.php:65]: That Oauth2Client is wrong. It should be OAuth2Client.

File: aqua/extraAquaSettings.php [hunk 3]
```diff
@@ -33,10 +33,9 @@ function ModifyDefaultLogin(&$personal_urls, &$wgTitle) {
 
 $wgHooks['PersonalUrls'][] = 'ModifyDefaultLogin';
 
-# Disable reading by anonymous users
-# WARNING: If you uncomment this line, remote verification via API will be
-# denied access.
-# $wgGroupPermissions['*']['read'] = false;
+# Disable reading by anonymous users using `./pkc setup --private -w WALLET_ADDRESS`
+# WARNING: remote verification via API will be denied access.
```
Comment (rht) [inline aqua/extraAquaSettings.php:37]: This warning doesn't explain whether the true/false state denies the API access. Needs to start with "If false, ...".

**Pull request merged**